pip/commands/install.py

```diff
@@ -297,6 +303,24 @@ def run(self, options, args):
                     installed = ' '.join(items)
                     if installed:
                         logger.info('Successfully installed %s', installed)
+                except EnvironmentError as e:
+                    message_parts = []
+
+                    user_option_part = "Consider using the `--user` option"
+                    permissions_part = "Check the permissions"
+
+                    if e.errno == errno.EPERM:
+                        if not options.use_user_site:
+                            message_parts.extend([
+                                user_option_part, " or ",
+                                permissions_part.lower(),
+                            ])
+                        else:
+                            message_parts.append(permissions_part)
+                        message_parts.append("\n")
+
+                    logger.error("".join(message_parts))
```

Comment:
In one of the reviews, @xavfernandez asked about having ``exc_info=True`` adding to this if ``options.verbose`` is high enough. I think that is a great idea and I'd like to see that here before this gets merged.